ui/admin/app/templates/application.hbs

```diff
@@ -79,23 +65,23 @@
               as |dropdown|
             >
               <dropdown.section @title={{t 'actions.toggle-feature-edition'}}>
-                <Rose::Form as |form|>
-                  <form.radioGroup
-                    @name='edition'
-                    @selectedValue={{this.featureEdition.edition}}
-                    @changed={{this.toggleEdition}}
-                    as |radioGroup|
-                  >
-                    {{#each this.featureEdition.editions as |edition|}}
-                      <dropdown.item>
-                        <radioGroup.radio
-                          @label={{edition}}
-                          @value={{edition}}
-                        />
-                      </dropdown.item>
-                    {{/each}}
-                  </form.radioGroup>
-                </Rose::Form>
+                <Hds::Form::Radio::Group
+                  @name='edition'
+                  class='theme-selector'
```

Comment:
Non-blocking: I see `theme-selector` class is being used for the theme and edition. Could we rename it to something more generic?

Reply:
updated!